decorator.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'nameable'
+
+class Decorator < Nameable
+  def initialize(nameable)
```

Comment:
- [ ] Kindly consider adding `attr_accessor :nameable` to allow access to instance variables from outside the scope of the class definition